superset/reports/commands/execute.py

```diff
@@ -656,6 +656,7 @@ def run(self) -> None:
                 state_found = True
                 break
         if not state_found:
+            logger.error('Report state not found, execution id: %s', self._execution_id)
```

Comment:
This will likely be a duplicate of the log that we will get when it raises below.